src/Symfony/Component/Notifier/Bridge/LinkedIn/LinkedInTransportFactory.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LinkedIn;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Smaïne Milianni <contact@smaine.me>
+ *
+ * @experimental in 5.2
+ */
+class LinkedInTransportFactory extends AbstractTransportFactory
+{
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+        $authToken = $this->getUser($dsn);
+        $accountId = $this->getPassword($dsn);
+        $host = 'default' === $dsn->getHost() ? null : $dsn->getHost();
+        $port = $dsn->getPort();
```

Comment:
Useless variables. IMHO

Reply:
copied from https://github.com/symfony/slack-notifier/blob/master/SlackTransportFactory.php#L30

Reply:
Yes, but those variables used only once.

Reply:
In my opinion, we have to have normalize code between bridges. In the other bridges, variables are declared and are used only once. However if you want to open a PR to edit this, you can @SHTIKOV.

Reply:
It's just IMHO :)